website/src/components/pageStateSelectors/WasapPageStateSelector.tsx

```diff
@@ -277,6 +278,20 @@ function ResistanceMutationsFilter({
     );
 }
 
+function KnownVariantsExclusionInfo() {
+    return (
+        <div className='relative p-8'>
+            <form method='dialog'>
+                <button className='btn btn-sm btn-circle btn-ghost absolute top-2 right-2'>✕</button>
+            </form>
+            <h1 className='mb-2 text-xl font-semibold'>How it works</h1>
```

Comment:
(We also have a `ModalHeader` component somewhere, but no need to change this now)